### PR TITLE
Updates some lavaland ruins and adds new ones.

### PR DESCRIPTION
**Pull request by lordpidey**
Adds lavaland wrath ruin.
Adds lavaland wrath ruin's reward.
Lavaland ruins can now cause sintouching.

I'm still working on adding a ruin for Acedia, but I wanted to run wrath's by the coderbase in the meantime.

Bottom line: wrath has an altar of wrath.  Upon using the altar, you will be equipped with a NODROP hand of wrath, which is a one time use melee weapon that gibs you, AND your target.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
>Adds lavaland wrath ruin.

Hm, guess I need a new plan for lavaland russians

File: tgstation.dme [hunk 1]
```diff
@@ -15,6 +15,7 @@
 // BEGIN_INCLUDE
 #include "_maps\__MAP_DEFINES.dm"
 #include "_maps\tgstation2.dm"
+#include "_maps\RandomRuins\LavaRuins\lavaland_surface_wrath.dmm"
```
Comment (coiax) [inline tgstation.dme]: Remove this.

File: code/modules/ruins/lavaland_ruin_code.dm [hunk 2]
```diff
@@ -178,13 +195,56 @@
...
+	icon_state = "disintegrate"
+	item_state = "disintegrate"
+	flags = NODROP
+	unacidable = 1
+
+/obj/item/weapon/melee/wrathhand/afterattack(atom/movable/AM, mob/living/carbon/human/user, proximity)
+	if(AM == user)
+		user << "<span class='warning'>Surely there is someone you hate more than yourself.</span>"
+	else
+		if(istype(AM,/mob/living))
```
Comment (coiax) [inline code/modules/ruins/lavaland_ruin_code.dm]: You need to check for the proximity flag otherwise it's going to be a ranged gib

**Comment by Bawhoppen** [comment]
do you have original assets for this ruin

**Comment by Man-Shroom-** [comment]
acedia is so stupid

lust is the seventh sin

**Comment by Zilenan91** [comment]
Why would russians be a wrath ruin? They could just be their own thing like ashwalker nests rather than a deadly sin.

**Comment by KorPhaeron** [comment]
They are raiding the wrath temple

**Comment by lordpidey** [comment]
@Bawhoppen No, I don't have original assets for either ruin.  I'd like some, but my art is terrible.

@Man-Shroom- I removed lust as a sin because no ERP.  So I had to look for alternatives in the historical seven deadly sins, and that left vainglory and acedia.  Acedia seemed like a better sin to include, so I put it in.

**Comment by Armhulen** [comment]
well i can see the reasoning in that, vainglory is a 4/5 moba

**Comment by lordpidey** [comment]
oh crap...
I forgot to put a DNM on this.

**Pull request merged**